index.html

```diff
@@ -0,0 +1,526 @@
+          <p class="text -small -color-dark">We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<cite><a href="">5</a></cite> Furthermore, adherence studies using text reminders show promising results.<cite><a href="">6</a></cite> Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<cite><a href="">7</a></cite></p>
+          <p class="text -small -color-dark">Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?</p>
+        </div>
+      </section>
+    </div>
+  </div>
+  <!--1) GREEN BACKGROUND END-->
+
+
+  <!--2) BLUE BACKGROUND START-->
+    <div class="bg-blue">
+      <!--MEET FLO SECTION-->
+      <div class="content-wrap">
+        <div class="flo-img"></div>
+        <section class="meet-flo">
+          <div class="centered">
+            <p class="text -x-small -color-white -caps -wide-spacing -bold">The Concept</p>
+            <p class="text -large -serif -color-white">Meet Florence</p>
+          </div>
+          <div class="narrow-padding">
+            <p class="text -small -color-white">Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen. A chatbot at heart, patients would converse with her through text and picture messages.</p>
+            <p class="text -small -color-white">If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.</p>
+          </div>
+        </section>
+      </div>
+    </div>
+  <!--2) BLUE BACKGROUND END-->
+
+
+  <!--3) GRADIENT BACKGROUND START-->
+    <div class="bg-gradient">
+      <!--ABOUT FLO SECTION-->
+      <section class="conversation-card content-wrap -cup">
+        <div class="about-flo-icon__bg"></div>
+        <div class="about-flo-icon"></div>
+        <div class="narrow-padding about-flo">
+          <div class="-content">
+            <p class="text -medium -color-lightblue -caps -bold centered">She gets to know you</p>
+            <p class="text -small -serif">Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+            <p class="text -small -serif">This conversation, which unfolds over time, feels less like twenty questions and more like the start of a relationship.</p>
+          </div>
+          <div class="-chat">
```

Comment:
Extra bonus credit: this could be fun to work through using `aria-live="polite"` and javascript to create a similar chat simulation for screen readers.